spec/lib/dttp/origin_page_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe OriginPage do
```

Comment:
Looks like this spec needs to be shifted one up from the `spec/lib/dttp` folder.

Reply:
Good spot!